src/components/table/table.tsx

```diff
@@ -4,6 +4,7 @@ import classNames from 'classnames/bind';
 import { ArrowDownIcon, ArrowUpIcon } from '@components/icons';
 import { FixedColumn, PrimaryColumn, RowData, TableComponentProps, SortDirection } from './types';
 import { Checkbox } from '@components/checkbox';
+import { getColumnsKeys } from '@components/table/utils';
```

Comment:
As the utils are Table internal utils, I suggest to use relative import here `./utils`.